tools/pysa_integration_tests/run.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import logging
+import os
+from pathlib import Path
+
+from utils import run_pysa_integration_test, LOG
+
+
+def main(
+    run_directory,
+    filter_issues,
+    skip_model_verification,
+    run_from_source,
+    passthrough_args,
+) -> None:
+    """
+    Entry point function which checks if full_result.json is there, calls
+    functions from integration_test_utils to run pysa, parse
+    full_result.json, and compare the output.
+    """
+    logging.basicConfig(
+        level=logging.DEBUG,
+        format="%(asctime)s %(levelname)s %(message)s",
+    )
+
+    full_result_file_path = run_directory / "full_result.json"
+
+    if not os.path.isfile(full_result_file_path):
+        raise FileNotFoundError(
+            f"{full_result_file_path} containing expected issues is not found"
+        )
+
+    LOG.info("Running in `%s`", run_directory)
+    run_pysa_integration_test(
+        run_directory,
+        passthrough_args=passthrough_args,
+        skip_model_verification=skip_model_verification,
+        filter_issues=filter_issues,
+        run_from_source=run_from_source,
+    )
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Run integeration tests")
+
+    parser.add_argument("--run-directory", type=Path)
+    parser.add_argument("--filter-issues", action="store_true")
+    parser.add_argument("--skip-model-verification", action="store_true")
+    parser.add_argument("--run-from-source", action="store_true")
+    parser.add_argument("--passthrough-args", nargs="+")
+
+    parsed = parser.parse_args()
+
+    run_directory = parsed.run_directory
+    if run_directory is None:
+        run_directory = Path(os.getcwd())
+    filter_issues = parsed.filter_issues
+    skip_model_verification = parsed.skip_model_verification
+    run_from_source = parsed.run_from_source
+    passthrough_args = parsed.passthrough_args
+    if passthrough_args is None:
+        passthrough_args = ""
```

Comment:
run_pysa_integration_test expects a `Sequence[str]`, so `passthrough_args` should be an empty list here.
I'm actually wondering if `parsed.passthrough_args` is not already an empty list if that argument is not provided.

Reply:
When its not specified I get a `None`